six/projects/csm/include/six/csm/SIXSensorModel.h

```diff
@@ -362,10 +362,11 @@ class SIXSensorModel : public csm::RasterGM
 public: // RasterGM methods
     /**
     * Returns the starting coordinate for the imaging operation.
-    *
-    * \return Always returns (0, 0)
+    * In case you have a chipped image, this will return the pixel
+    * offset to the chip.
+    * \return starting coordinate for the imaging operation
     */
-   virtual csm::ImageCoord getImageStart() const;
+   virtual csm::ImageCoord getImageStart() const = 0;
```

Comment:
This was already declared pure virtual in `RasterGM`.  So, it can be simply removed here.